src/store/state/resource-model.ts

```diff
@@ -69,8 +69,34 @@ const dissolutionResourceModel: DissolutionResourceIF = {
   custodialRecords: null
 }
 
+const restorationResourceModel: RestorationResourceIF = {
+  entityType: null,
+  displayName: '',
+  steps: [],
+  filingData: null,
+  reviewAndConfirm: {
+    completingPartyStatement: {
+      certifyStatementHeader: null,
+      certifyStatements: [],
+      certifyClause: null,
+      entityDisplay: null
+    }
+  },
+  peopleAndRoles: {
+    header: null,
+    blurb: null,
+    helpSection: null,
+    blurb2: null,
+    addIncorporator: null,
+    addOrganization: null,
+    addBusiness: null,
+    rules: []
+  }
+}
```

Comment:
To reduce duplicate code and keep things consistent, we should have an "empty people and roles" object and an empty "review and confirm" object that we can just spread or clone here.

You don't have to do this in this ticket -- we should have a refactor / tech debt ticket to clean up some of this kind of stuff.

Reply:
I can add a ticket for this.

Reply:
Yes, please and thanks!